llm/server.go

```diff
@@ -81,6 +81,46 @@ func LoadModel(model string, maxArraySize int) (*GGML, error) {
 	return ggml, err
 }
```

Comment:
From some comments in the PR and our discussion, it seems we may need to disable KV cache quantization for embedding models.

We do this today by looking for the pooling layer (i.e. a `<architecture>.pooling_type` KV in the GGUF file) in `images.go`: https://github.com/ollama/ollama/blob/main/server/images.go#L99

Reply:
Oohhh that's nice! I've now added a small function to check for that in server.go.